site_root/source/resources/StructureDefinition-Phenotype.json

```diff
@@ -0,0 +1,36 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "Phenotype",
+  "url": "http://fhir.kids-first.io/StructureDefinition/Phenotype",
+  "name": "Phenotype",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Observation",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Observation",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+        {
+            "id": "Observation.extension",
+            "path": "Observation.extension",
+            "slicing": {
```

Comment:
I think the Type.extension element has an implicit slicing on the url and there shouldn't be a need to declare one. Also, I don't see the slice for the extension.

Reply:
Ok let me remove the `slicing`. Ah I think I might have forgot to add the slice...wonder how that passed validation.